drivers/dma/dma_mcux_lpc.c

```diff
@@ -543,7 +586,16 @@ static int dma_mcux_lpc_configure(const struct device *dev, uint32_t channel,
 			assert(block_config->dest_address ==
 			       ROUND_UP(block_config->dest_address, width));
 
-			if (dma_mcux_lpc_queue_descriptors(data, block_config, src_inc, dst_inc)) {
+			if (block_config->next_block == NULL) {
+				/* This is the last block. If complete callback is called only
+				 * on the last block then enable it for the last block
```

Comment:
i am confused by the wording of this comment

Reply:
I have updated the comment.